modules/nf-core/mageck/count/main.nf

```diff
@@ -12,27 +12,35 @@ process MAGECK_COUNT {
     path(library)
 
     output:
-    tuple val(meta), path("*count.txt")           , emit: count
+    tuple val(meta), path("*count.txt"),             emit: count
     tuple val(meta), path("*.count_normalized.txt"), emit: norm
-    path "versions.yml"                            , emit: versions
+    tuple val(meta), path("*.countsummary.txt"),     emit: summary
```

Comment:
This one missing the underscore makes me sad, but I assume that is the tool's error...

Reply:
yes not much i can do about this one except express sympathy sorry :(